### PR TITLE
Add chat component

### PR DESCRIPTION
**Pull request by liannivin**

### CODE REVIEW

File: src/home.js [hunk 1]
```diff
@@ -125,10 +108,8 @@ function Home({ url }) {
           </header>
           <ListChannels />
           <Tabs />
-          <section id="js-messages-list" className="chat padding24">
-            <ListMessages message={listMessage} />
-            <SendMessage />
-          </section>
+          <section id="js-messages-list" className="chat padding24" />
+          <Chat />
```
Comment (cdsandoval) [inline src/home.js]: This should be like this
<section> 
  <Chat />
</section>

File: src/home.js [hunk 2]
```diff
@@ -125,10 +108,8 @@ function Home({ url }) {
           </header>
           <ListChannels />
           <Tabs />
-          <section id="js-messages-list" className="chat padding24">
-            <ListMessages message={listMessage} />
-            <SendMessage />
-          </section>
+          <section id="js-messages-list" className="chat padding24" />
```
Comment (cdsandoval) [inline src/home.js]: You must delete it if the component already have it.

File: src/components/chat/chat.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+function Chat() {
+  const [listMessage, setListMessage] = React.useState([]);
+  const [text, setText] = React.useState("");
+
+  function addMessage(event) {
+    event.preventDefault();
+    setListMessage([
+      ...listMessage,
+      { Author: "Carlos", text: text, date: new Date() }
```
Comment (yummta) [inline src/components/chat/chat.js]: Can you explain to me why you used "Carlos" as value for Author?

Comment (liannivin) [reply src/components/chat/chat.js]: We didn't use `local storage` to save data.

**Pull request merged**